### PR TITLE
Add data loaders

### PR DESCRIPTION
**Pull request by martbern**
General functions should be at the top of each class. I don't expect you to review every outcome definition ;-)

As the projects grow, we'll gain a more informed opinion about which outcomes to keep in a common module and which to have as project specific.

### CODE REVIEW

File: src/loaders/medications.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        view: str,
+        new_col_str: str = None,
+        depth: int = None,
+    ) -> pd.DataFrame:
+        """Load the prescribed medications that match atc from the beginning of their atc string.
+        Aggregates all that match. Beware that data is incomplete prior to sep. 2016 for prescribed medications.
+
+        Args:
+            atc_str (str): ATC string to match on.
+            source_timestamp_col_name (str): Name of the timestamp column in the SQL table.
```
Comment (KennethEnevoldsen) [inline src/loaders/medications.py:79]: I prefer name instead of str

File: src/loaders/diagnoses.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        df["val"] = 1
+
+        df.rename(columns={"datotid_first_t2d_diagnosis": "timestamp"}, inplace=True)
+
+        msg.good("Finished loading t2d event times")
+        return df.reset_index(drop=True)
+
+    def essential_hypertension():
+        return LoadDiagnoses.from_physical_visits(
+            icd_str="I109", wildcard_icd_10_end=False
```
Comment (HLasse) [inline src/loaders/diagnoses.py:160]: add col name for readability/interpretability later

File: src/loaders/diagnoses.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        return LoadDiagnoses.from_physical_visits(
+            icd_str=[
+                "E780",
+                "E785",
+            ],  # Only these two, as the others are exceedingly rare
+            new_col_str="hyperlipidemia",
+            wildcard_icd_10_end=False,
+        )
+
+    def liverdisease_uns():
```
Comment (HLasse) [inline src/loaders/diagnoses.py:173]: add col name for readability/interpretability later

File: src/loaders/sql_load.py [hunk 4]
```diff
@@ -1,14 +1,11 @@
...
-
-import time
 import urllib
 import urllib.parse
-
 from typing import Generator, Optional, Union
 
+import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.pool import NullPool
```
Comment (KennethEnevoldsen) [inline src/loaders/sql_load.py:7]: Check if it is used

File: src/loaders/str_utils.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+from pandas import DataFrame
+from wasabi import msg
+
+
+def create_cols_for_unique_vals_at_depth(
```
Comment (KennethEnevoldsen) [inline src/loaders/str_utils.py:5]:
```suggestion
def create_cols_for_unique_values_at_depth(
```

**Review by HLasse — CHANGES_REQUESTED** [review]

File: src/loaders/diagnoses.py (hunk 3 above)
Comment (HLasse) [reply src/loaders/diagnoses.py:173]: also goes for the others further down..

File: src/loaders/demographics.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from wasabi import msg
+
+from loaders.sql_load import sql_load
+
+
+class LoadDemographics:
+    def birthdays():
```
[the remark this answers is not in the document]
Comment (martbern) [reply src/loaders/demographics.py:8]: #68

**Pull request merged**